magefile.go

```diff
@@ -109,3 +128,76 @@ func makeTestTask(args ...string) func(ctx context.Context) error {
 		return sh.RunV("ginkgo", append([]string{"-p"}, args...)...)
 	}
 }
+
+func LegacyTestSuiteTags(ctx context.Context) error {
```

Comment:
perhaps we should
- run the tests against every version?
- stop tightly coupling test suites and the cli config format (See `withCaptainConfig`)

Reply:
discussed with @kylekthompson. going to write the config as yaml and continue the strategy here